src/main/java/rssreader/factory/DtoFactory.java

```diff
@@ -0,0 +1,55 @@
+package rssreader.factory;
+
+import rssreader.dto.RssDto;
+import rssreader.dto.RssFeedDto;
+import rssreader.entity.RssEntity;
+import rssreader.entity.RssFeedEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DtoFactory {
+    private static DtoFactory instance = null;
+
+    public static synchronized DtoFactory getInstance(){
+        if(instance == null){
+            instance = new DtoFactory();
+        }
+        return instance;
+    }
+
+    public RssDto getRssDto(RssEntity entity){
```

Comment:
Method name is not informative enought. In fact it creates newDTO object, Right? So, call it something like: createRssDto.